.dockerignore

```diff
@@ -1,3 +1,6 @@
 **/.gitignore
 **/node_modules
-**/Dockerfile*
\ No newline at end of file
+**/Dockerfile*
+**/openshift
+**/junk
```

Comment:
Do we want to include junk in this list? I presume it's a Sy-specific local folder?

Reply:
I'm happy to rename my junk folder to something that seems less Sy specific - but I do need a place to put the many gigs of data I seem to accumulate.